l10n_co_res_partner/data/l10n_states_co_data.xml

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<openerp>
```

Comment:
Can you refer to a link with the proper refence of this data, it is usefull for future erferences.